src/main/java/ch/uzh/ifi/hase/soprafs24/controller/SpotifyController.java

```diff
@@ -1,4 +1,34 @@
 package ch.uzh.ifi.hase.soprafs24.controller;
 
+import ch.uzh.ifi.hase.soprafs24.rest.dto.PlaylistCollectionDTO;
+import ch.uzh.ifi.hase.soprafs24.service.SpotifyService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/user/playlist")
 public class SpotifyController {
+
+    @GetMapping("/names")
+    public ResponseEntity<PlaylistCollectionDTO> getSpotifyUserId(@RequestParam String accessToken) {
+        System.out.println(accessToken);
+        // Call the service method and return its response
+        PlaylistCollectionDTO dto = SpotifyService.getUserPlaylistNames(accessToken);
+
+        if (dto != null) {
```

Comment:
I think this block can be simplified:
lets just set the default return type to OK 

and in the service handle the edge cases